region/maridia/outer/Mt. Everest.json

```diff
@@ -1190,6 +1191,24 @@
       ],
       "devNote": "Although this may go through 7 and 8, it requires starting at 2 to get the crab to move into position."
     },
+    {
+      "link": [2, 11],
+      "name": "Cross Room Jump - Tricky Dash Jump",
+      "requires": [
+        "canCrossRoomJumpIntoWater",
+        {"adjacentJumpway": {
+          "jumpwayType": "platformBelow",
+          "fromNode": 2,
+          "maxHeight": 6,
+          "maxLeftPosition": -2.5,
+          "minRightPosition": 2.5
```

Comment:
```suggestion
          "maxLeftPosition": 2.5,
          "minRightPosition": -2.5
```
Switching the signs makes the strat not depend on jumpway length, which I think is what is intended. Otherwise it would require the jumpway to contain all four tiles below the door, which doesn't appear to be needed given that it only uses 1 tile of runway.